.github/workflows/main.yml

```diff
@@ -24,5 +24,5 @@ jobs:
       - name: Build & Test
```

Comment:
@ijnek @mossmaurice 
One way could be to remove the iceoryx packages that come together with ROS Rolling to enforce using the custom built iceoryx version.

Something like this:
```console
sudo dpkg -r --force-depends "ros-rolling-iceoryx-posh"
```
would remove the previously installed debian package without removing dependencies like `rmw_cyclonedds_cpp`

The other option is to make a complete clean build from source (https://github.com/ros-tooling/setup-ros#setting-up-the-worker-and-installing-the-system-dependencies).